client/my-sites/stats/activity-log/index.jsx

```diff
@@ -605,7 +605,16 @@ class ActivityLog extends Component {
 				<StatsFirstView />
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
-				{ 'awaitingCredentials' === rewindState.state && <ActivityLogCredentialsNotice /> }
+				{ 'awaitingCredentials' === rewindState.state && (
+					<Banner
+						icon="history"
+						href={ `/settings/security/${ slug }` }
+						title={ translate( 'Add your site credentials' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Add site credentials' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).